src/ert/gui/tools/run_analysis/run_analysis_tool.py

```diff
@@ -177,11 +177,7 @@ def _enable_dialog(self, enable: bool):
             QApplication.setOverrideCursor(Qt.CursorShape.WaitCursor)
 
     def _init_analyse(self, source_fs: EnsembleReader, target: str):
-        with add_context_to_error(
```

Comment:
Why remove the context?

Reply:
There is no more `to_accessor` function because this PR removes the Reader/Accessor distinction. This error can't be triggered anymore, so we don't need to add context to an error that can't occur.